boa/src/syntax/parser/expression/assignment/arrow_function.rs

```diff
@@ -71,7 +71,12 @@ where
     fn parse(self, cursor: &mut Cursor<R>) -> Result<Self::Output, ParseError> {
         let _timer = BoaProfiler::global().start_event("ArrowFunction", "Parsing");
 
+        println!("Parsing arrow func");
+
         let next_token = cursor.peek(0)?.ok_or(ParseError::AbruptEnd)?;
+
+        println!("Next token {}", next_token);
+
```

Comment:
It seems there are some debugging statements left.

Reply:
Ah yes - removed